examples/Button.json

```diff
@@ -127,7 +127,39 @@
         }
     ],
     "computed": [],
-    "methods": [],
+    "methods": [
+        {
+            "keywords": [
+                {
+                    "name": "param",
+                    "description": "{string} question a question about life, the universe, everything"
+                },
+                {
+                    "name": "returns",
+                    "description": "{number} the answer to all your questions"
+                }
+            ],
+            "visibility": "public",
+            "description": "Computes the answer to your question.",
+            "name": "computeAnswer",
+            "args": [
+                {
+                    "name": "question"
+                }
+            ],
+            "return": {
+                "doctype": {
```

Comment:
This field must be called a `type` instead of `doctype`, like it used everywhere